docs/src/fs.rst

```diff
@@ -218,7 +218,8 @@ API
 
 .. c:function:: int uv_fs_read(uv_loop_t* loop, uv_fs_t* req, uv_file file, const uv_buf_t bufs[], unsigned int nbufs, int64_t offset, uv_fs_cb cb)
 
-    Equivalent to :man:`preadv(2)`.
+    Equivalent to :man:`preadv(2)`. One exception is that if you use an `offset` of `-1`, then it reads from the current file
+    position instead of first seeking.
```

Comment:
This might be simpler?
```suggestion
    Equivalent to calling :man:`preadv2(2)` with no flags set.
```

(style nit: need to line-wrap at 80 columns)

Reply:
preadv2 does follow the -1 rule (I did not notice this originally). However, it's a non-standard linux extension so I'm not sure the docs should point at it.

And also I would suggest that being explicit about the API is more effective than requiring people to find a small caveat in a man page that has 5 different functions described.

I will do the line wrap.

Reply:
Updated, I just copied the note from the man page.